routers/api/v1/swagger/activitypub.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package swagger
+
+import (
+	api "code.gitea.io/gitea/modules/structs"
+)
+
+// ActivityPub
+// swagger:response ActivityPub
+type swaggerResponseActivityPub struct {
+	// in:body
```

Comment:
Given that this seems to be intended for machines:
```suggestion
//swagger:response ActivityPub
type swaggerResponseActivityPub struct {
	//in:body
```

Reply:
All of the other files in that directory have comments with the leading space, so is this really needed?

Reply:
According to my knowledge of Go, comments intended for machines have no leading space, while those for humans have one.
If it currently works as intended, that's rather a courtesy of Go than intention.
So yeah, according to the convention, yes.
But I'll leave the choice up to you.

Reply:
To note here, feel free to enforce this via: https://gitea.com/gitea/gitea-vet consistency is more important than small change here.